model-optimizer/mo/utils/ir_reader/extenders/deconvolution_extender.py

```diff
@@ -25,6 +25,9 @@ def extend(op: Node):
 
 
 def common_backpropdata_extender(op: Node):
+    for attr in ['strides', 'output_padding', 'pads_begin', 'pads_end', 'dilations']:
```

Comment:
why is `auto_pad` not listed here?

Reply:
`auto_pad` is a `string` value, no need to convert it to `list`

Reply:
Yes, we need to handle only `list` attributes using this function.